web/themes/custom/novel/novel.theme

```diff
@@ -109,6 +109,7 @@ function novel_preprocess_dpl_react_app__search_result(array &$variables): void
 function novel_preprocess(array &$variables): void {
   $directory = $variables['directory'] ?? 'web/themes/custom/novel';
   $variables['baseIconPath'] = $directory . '/assets/dpl-design-system/icons';
+  $variables['absoluteBaseUrl'] = \Drupal::request()->getSchemeAndHttpHost();
```

Comment:
isnt it a bit extreme to add this to **every** preprocess there is?
wouldnt it be better to just add it to e.g. `novel_preprocess_paragraph`

I do know that you're adding this to the existing hook, but atleast that is really simple code (e.g. based on static strings)

Reply:
I'm fine with moving it to paragraph preprocess instead 👍